README.md

```diff
@@ -86,46 +88,49 @@ Then, the geth console can start up and you can start a miner as follows:
 Now, save the passphrase you used in the geth node into a text file called password.txt. Then, once you have this private geth node running on your local network, we will need to generate test, pending transactions that can then be processed into collations by proposers. For this, we have created an in-house transaction generator CLI tool.
 
 
-# Sharding Minimal Protocol
+# Running Ethereum 2.0
 
-**NOTE**: This section is in flux: will be deprecated in favor of a beacon chain)
+**NOTE**: This section is in flux, much of this will likely change as the beacon chain spec evolves.
 
 Build our system first
 
 ```
-$ bazel build //client/...
+bazel build //...
 ```
 
-## Becoming a Attester
+## Step 1: Deploy a Validator Registation Contract
+
+Deploy the Validator Registration Contract into the chain of the running geth node by following the instructions [here](https://github.com/prysmaticlabs/prysm/blob/master/contracts/validator-registration-contract/deployVRC/README.md).
+
+## Step 2: Running a Beacon Node
+
+Make sure a geth node is running as a separate process according to the instructions from the previous section. Then, you can run a full beacon node as follows:
+
+```
+./bazel-bin/beacon-chain/YOUR-ARCHITECTURE/beacon-chain --web3provider ws://127.0.0.1:8546 --datadir /path/to/your/datadir --rpc-port 5000
+```
 
+This will spin up a full beacon node that connects to your running geth node, opens up an RPC connection for sharding clients to connect to it, and begins listening for p2p events.
 
-Make sure a geth node is running as a separate process. Then, to deposit ETH and join as a attester in the Sharding Manager Contract, run the following command:
+To try out the beacon node in development by simulating incoming blocks, run the same command above but enable the `--simulator` and a debug level, log verbosity with `--verbosity debug` to see everything happening underneath the hood.
 
 ```
-bazel run //client -- \
-   --actor "attester" \
-   --deposit \
-   --datadir /path/to/your/datadir \
-   --password /path/to/your/password.txt \
-   --networkid 12345
+./bazel-bin/beacon-chain/YOUR-ARCHITECTURE/beacon-chain --web3provider ws://127.0.0.1:8546 --datadir /path/to/your/datadir --rpc-port 5000 --simulator --verbosity debug
```

Comment:
Can you change this to use `bazel run` and format it so that there is 1 flag per line? It helps for readability. 

The concern about multiple `bazel run`s at the same time is no longer an issue.